packages/react/src/FilteredActionList/FilteredActionListWithModernActionList.tsx

```diff
@@ -206,14 +155,28 @@ export function FilteredActionList({
                       </ActionList.GroupHeading>
                       {getItemListForEachGroup(group.groupId).map((item, index) => {
                         const key = item.key ?? item.id?.toString() ?? index.toString()
-                        return <MappedActionListItem key={key} {...item} renderItem={listProps.renderItem} />
+                        return (
+                          <MappedActionListItem
+                            key={key}
+                            {...item}
+                            renderItem={listProps.renderItem}
+                            data-select-panel-item="true"
```

Comment:
should this also be dependent on the context container?

Reply:
Ah, I think we can just remove this because we're adding it inside `MappedActionListItem` and doing the appropriate check.